widget/entry.go

```diff
@@ -426,25 +426,41 @@ func (e *Entry) Focused() bool {
 func (e *Entry) cursorColAt(text []rune, pos fyne.Position) int {
 	for i := 0; i < len(text); i++ {
 		str := string(text[0 : i+1])
-		wid := textMinSize(str, theme.TextSize(), e.textStyle()).Width
+		wid := textMinSize(str, theme.TextSize(), e.textStyle()).Width + theme.Padding()
 		if wid > pos.X {
 			return i
 		}
 	}
 	return len(text)
 }
 
-// Tapped is called when this entry has been tapped so we should update the cursor position.
-func (e *Entry) Tapped(ev *fyne.PointEvent) {
```

Comment:
By moving from Tapped to MouseDown you are relying on desktop extensions for any tap handling in this widget.
We can't do that or future mobile/web support will break.
You can add MouseDown but cannot remove Tapped...

Reply:
I see. I'll take this back to the drawing board.